scripts/validate_public_release.py

```
@@ -504,42 +510,62 @@ def canonical_public_release_validator_path(name: str) -> str:
+
+def _load_public_release_package_catalog(root: Path) -> dict[str, tuple[str, ...]]:
+    """Load the catalog that authorizes package-owned release registrations."""
+
+    path = root / PUBLIC_RELEASE_PACKAGE_CATALOG_PATH
     try:
         metadata = path.lstat()
     except FileNotFoundError as error:
         raise ReleaseError(
-            "public-release runtime-package catalog is missing"
+            "public-release package registration catalog is missing"
         ) from error
```

Ruff (0.16.3)

[warning] Avoid specifying long messages outside the exception class

(TRY003)